tests/backup_restore_test.go

```diff
@@ -10,10 +10,11 @@ import (
 
 	"fmt"
 
+	"strings"
```

Comment:
Can you please manually fix up the spacings? Our usual standard is two groups, stdlib and everything else.

`goimports` doesn't enforce how many groups we have, just that the groups are internally sorted.

Reply:
You can ignore anything autogenerated - ideally we would exclude autogenerated code from `goimports` but we haven't done that yet.